Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM dmstr/php-yii2:7.0-fpm-1.9-beta2-alpine-nginx
```

Comment:
This would imply that we use another way of getting the framework source-code onto the image, because a Dockerfile defines the build context and you can not add something from outside the build context.

An option is to use host-volumes, but they can problematic regarding performance and other things.
We could also clone or download the current commit when building an image, but that would be annoying in development.

I also don't like it, that these files are in the root folder, but it was the best option to me for now.

I am open for discussion.